src/github.com/t3chguy/matrix-static/templates/room-chat.qtpl

```diff
@@ -199,7 +199,7 @@
         {% if memberInfo.AvatarURL.IsValid() %}
             <img class="avatar userAvatar" src="{%s memberInfo.AvatarURL.ToThumbURL(48, 48, "crop") %}" />
         {% else %}
-            <img class="avatar userAvatar" src="./img/logo_missing.png" />
+            <img class="avatar userAvatar" src="./avatar/{%u memberInfo.GetName() %}" />
```

Comment:
`/avatar/` does no fs access
for the css static fs route, gin cancels out any `../` and sends a redirect before processing it'd seem